TestFoundation/TestNSJSONSerialization.swift

```diff
@@ -1054,6 +1058,26 @@ extension TestNSJSONSerialization {
         XCTAssertEqual(try trySerialize(json), "[false,true]")
     }
     
+    func test_serialize_IntMax() {
+        let json: [Any] = [Int.max]
```

Comment:
Yes. [Documentation](https://developer.apple.com/reference/swift/int):
> On 32-bit platforms, Int is the same size as Int32, and on 64-bit platforms, Int is the same size as Int64.

So I think this will break on 32-bit platforms.  We should construct the string we're asserting against using the same constant, I think.  The same applies for the tests which use `Int.min`, `UInt.max` and `UInt.min`.